tony-core/src/main/java/com/linkedin/tony/util/Utils.java

```diff
@@ -293,6 +293,8 @@ public static int executeShell(String taskCommand, long timeout, Map<String, Str
     ProcessBuilder taskProcessBuilder = new ProcessBuilder("bash", "-c", taskCommand);
     taskProcessBuilder.redirectError(ProcessBuilder.Redirect.INHERIT);
     taskProcessBuilder.redirectOutput(ProcessBuilder.Redirect.INHERIT);
+    // Unset MALLOC_ARENA_MAX for better performance
+    taskProcessBuilder.environment().remove("MALLOC_ARENA_MAX");
```

Comment:
Can you include a link to https://github.com/linkedin/TonY/issues/346 as a reference in your code comment?

Reply:
Sure, I just pushed a new commit.